v0.0.1/admin-schema.json

```diff
@@ -0,0 +1,633 @@
+        "schema_version": {
+            "type": "string",
+            "format": "uri"
+        },
+        "name": {
+            "description": "The name of the hub.",
+            "type": "string",
+            "examples": "US COVID-19 Forecast Hub​"
+        },
+        "maintainer": {
+            "description": "The entity that maintains and runs the hub.",
+            "type": "string",
+            "examples": "The Consortium of Infectious Disease Modeling Hubs"
+        },
+        "contact": {
+            "description": "The name and email of a human being who serves as a point of contact for the hub.",
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string"
+                },
+                "email": {
+                    "type": "string",
+                    "format": "email"
+                }
+            }
+        },
+        "repository_host": {
+            "description": "The name of the web host for the hub repository",
+            "type": "string",
+            "enum": [
+                "github"
+            ]
+        },
+        "repository_url": {
+            "description": "The url for the hub repository.",
+            "type": "string",
+            "examples": "https://github.com/reichlab/covid19-forecast-hub"
+        },
+        "zoltar_project_id": {
+            "description": "The project id of the Hub in Zoltar",
+            "type": "integer",
+            "examples": 44
+        },
+        "hub_models": {
+            "description": "Array of ensemble and baseline models produced by the hub",
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "team_abbr": {
+                        "description": "Abbreviated team name of the hub",
+                        "type": "string",
+                        "pattern": "^[a-zA-Z0-9_+]+$",
+                        "maxLength": 16
+                    },
+                    "model_abbr": {
+                        "description": "Abbreviated name of the model",
+                        "type": "string",
+                        "pattern": "^[a-zA-Z0-9_+]+$",
+                        "maxLength": 16
+                    },
+                    "model_type": {
+                        "description": "The type of model: baseline or ensemble",
+                        "type": "string",
+                        "enum": [
+                            "baseline",
+                            "ensemble"
+                        ]
+                    }
+                }
+            }
+        },
+        "citation": {
+            "description": "One or more citations for the hub.",
+            "type": "string",
+            "examples": "Cramer, E.Y., Huang, Y., Wang, Y. et al. The United States COVID-19 Forecast Hub dataset. Sci Data 9, 462 (2022). https://doi.org/10.1038/s41597-022-01517-w"
```

Comment:
Noting that the "examples" do not show up on the docson. is there something we can do about that?

Reply:
If you click on the upper right of each element, it shows the underlying json which includes the examples but I agree, it's not ideal:
![image](https://user-images.githubusercontent.com/5583057/210764762-aa51fcf7-5118-433a-95b6-df05303aa008.png)